assets/lokomotive-kubernetes/bootkube/resources/charts/bootstrap-secrets/values.yaml

```diff
@@ -0,0 +1,3 @@
+bootstrapTokens:
+- tokenID: ""
+  tokenSecret: ""
```

Comment:
```suggestion
bootstrapTokens: []
#- tokenID: ""
#  tokenSecret: ""
```

This way, rendering the chart with no values provided will produce only valid objects. Also if we add validation rules, they should reject empty `tokenID` etc.

Reply:
What do you mean by `validation rules` ?

Reply:
It's possible to implement values validation rules in chart itself, using the templates. Maybe at some point it would make sense to add such and then empty `tokenID` would be an invalid value.